teuthology/__init__.py

```diff
@@ -14,10 +14,17 @@
 
 import logging
 import os
+import subprocess
 
+mydir = os.path.dirname(os.path.realpath(__file__))
 
-__version__ = '0.1.0'
-
+try:
+    __version__ = subprocess.check_output(
```

Comment:
I'm amenable.  Maybe we should call this 1.0.0 now since if this isn't production I don't know what is